builder-22/builder.toml

```
@@ -24,6 +24,10 @@ version = "0.14.1"
   id = "heroku/java-function"
   uri = "docker://public.ecr.aws/heroku-buildpacks/heroku-java-function-buildpack@sha256:46e353cadcd941baeb0f566d7ab7b1f13c18e96223c1143dc51b258638ab9d47"
 
+[[buildpacks]]
+  id = "heroku/go"
+  uri = "docker.io/heroku/buildpack-go@sha256:fdc270c4414dc636daf29e269a67f28bb1a8c4aee89974d80b90edbdc10ae8a2"
```

(for parity with the other entries, and also since I think otherwise the Pack CLI uses fuzzy heuristics to work out what the URL refers to)

```suggestion
  uri = "docker://docker.io/heroku/buildpack-go@sha256:fdc270c4414dc636daf29e269a67f28bb1a8c4aee89974d80b90edbdc10ae8a2"
```